cfe_internal/enterprise/federation/federation.cf

```diff
@@ -195,16 +195,17 @@ bundle agent transport_user
       edit_line => default:insert_lines("IdentityFile $(ssh_priv_key)");
 
   commands:
+    selinux_enabled.missing_ssh_context::
+      "/usr/bin/chcon -v --type ssh_home_t $(home)/.ssh/"
```

Comment:
Would be better to add a new *fcontext* rule because this will be reverted by relabeling otherwise.